prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -1,6 +1,28 @@
 <div class="container">
   <app-banner></app-banner>
   <div class="row">
+    <app-alert *ngIf="loginCancelled"
+               type="info"
+               icon="error_outline">
+      <ng-container #alertTitle
+                    class="alert-title">
+        Login Cancelled
+      </ng-container>
+      <ng-container #alertContent
+                    class="alert-content">
+        <p>You have cancelled your login to PRIME</p>
+        <p>For assistance with PRIME, contact {{ primeSupportPhoneNumber | phone }} or
+          <a [attr.href]="'mailto:' + primeSupportEmail">{{ primeSupportEmail }}</a>
+        <p>
+          For assistance logging in with the BC Services Card app, contact the Helpdesk
+          <a
+              target="_blank"
+              rel="noopener noreferrer"
+              [attr.href]="bcscHelpDeskUrl">
+              here</a>.
```

Comment:
This here and the other clickable links are hard to make out in this alert. Can we see about making it more visible?

Reply:
agreed. underlined `here` to be more visible as the mockup.